Ajoute une section tutoriels dans l'espace développeurs (#1522)
etalab/data_pass: CLAUDE.md

- Be sure that tests you introduce pass before stopping your work.
- Authorization should be done in the controller, not in the model/services.
- Check docs/ for technical documentation if needed.
- When touching the API (endpoints, payloads, événements webhooks, scopes,
  politique de retry, etc.), iterate on the relevant files under `docs/` and on
  the tutorials (`docs/developers/tutorials/` + `app/views/developers/tutorials/`)

>   so docs et tutos restent alignés avec le code.

Reviewer: WTF c'est quoi ce franglais lol ?

Author: Oui ? C'est la mémoire de Claude, le mieux placé pour écrire et lire dedans c'est lui. Je ne touche quasiment jamais ces fichiers manuellement.

Author: Propose un patch, apply et merge si tu veux, sinon merge, c'est du détail qui ne mérite pas un nième A/R 🤷

Reviewer: "so docs et tutos restent alignés avec le code."

Author: J'ai lu. Cf mes coms plus haut 🤷

>   so docs et tutos restent alignés avec le code.

Reviewer: ```suggestion
  so docs et tutos remain consistent with the code.
```
